Makefile

```diff
@@ -461,12 +464,12 @@ out/docs/minikube.md: $(shell find "cmd") $(shell find "pkg/minikube/constants")
 	go run -ldflags="$(MINIKUBE_LDFLAGS)" -tags gendocs hack/help_text/gen_help_text.go
 
 deb_version:
-	@echo $(DEB_VERSION)
+	@echo $(DEB_VERSION)-$(DEB_REVISION)
```

Comment:
This breaks my test (then again, nothing else uses "deb_version" afaik...)

But it is now different from the rpm_version, so please revert this line ?

Reply:
which test?

The intent here is to make it so that we can st DEB_REVISION to a non-zero value if required.

Reply:
Maybe change `rpm_version` to match, then ?

Reply:
PTAL.